conf/application.conf

```diff
@@ -69,6 +69,12 @@ microservice {
   }
 
   services {
+    tracking-consent-frontend  {
+      protocol = "http"
+      host = "localhost"
+      port = 12345
+      path = "/tracking-consent/tracking.js"
```

Comment:
Maybe more descriptive key name, like tracking-script-path?

Reply:
What about script-path?

Reply:
I think that's better, yes.